soda/core/soda/soda_cloud/soda_cloud.py

```diff
@@ -185,14 +202,10 @@ def get_historic_data(self, historic_descriptor: HistoricDescriptor):
         return {"measurements": measurements, "check_results": check_results}
 
     def is_samples_disabled(self) -> bool:
```

Comment:
please give extra 👀 on this is_samples_disabled and how organization config is handled now - nothing popped out from local testing and tests and we do have a test for this https://github.com/sodadata/soda-core/blob/081386dc6ebf99457b623749529c1e6030359d04/soda/core/tests/data_source/test_samples.py#L34 but this would be pretty bad to have a bug introduced